location/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.contrib import admin
```

Comment:
@usmandap we dont need to create a new app for every model, location and sponser both can come under event app

Reply:
@usmandap ?